Demo/imdb/imdb_demo.py

```diff
@@ -1,196 +1,81 @@
-import os
 import sys
-import csv
 import redis
-import timeit
-from datetime import date
-from disposableredis import DisposableRedis
-from redisgraph import Node, Edge, Graph
+from redisgraph import Graph
 
-REDIS_MODULE_PATH_ENVVAR = 'REDIS_MODULE_PATH'
-REDIS_PATH_ENVVAR = 'REDIS_PATH'
-REDIS_PORT_ENVVAR = 'REDIS_PORT'
+sys.path.append('.')
```

Comment:
Is this really necessary?
Suppose imdb_utils is within the same folder as this file, can't we just import imdb_utils?

Reply:
I have tried that, and it threw an exception. the reason it happens is because in order for us to see the other files we need to have __init__.py files in the different folders, but it needs to start from the main folder, and I don't think we want to put a __init__.py file there because this is a c/c++ project.